package.json

```diff
@@ -43,41 +43,42 @@
     "moment": "^2.18.1",
     "morgan": "^1.8.1",
     "request": "^2.81.0",
-    "superagent": "^3.5.2"
+    "superagent": "^3.5.2",
+    "superagent-debugger": "^1.2.9"
   },
   "devDependencies": {
-    "@microsoft/microsoft-graph-types": "^0.4.1",
-    "@types/body-parser": "^1.16.3",
-    "@types/chai": "^3.0.0",
+    "@microsoft/microsoft-graph-types": "0.4.1",
+    "@types/body-parser": "1.16.3",
+    "@types/chai": "3.5.2",
     "@types/chai-as-promised": "0.0.30",
     "@types/config": "0.0.32",
-    "@types/cors": "^2.8.1",
-    "@types/dotenv": "^2.0.20",
-    "@types/express": "^4.0.35",
+    "@types/cors": "2.8.1",
+    "@types/dotenv": "2.0.20",
+    "@types/express": "4.0.35",
     "@types/log4js": "0.0.32",
-    "@types/mocha": "^2.0.0",
-    "@types/node": "^6.0.68",
+    "@types/mocha": "2.2.41",
+    "@types/node": "7.0.22",
     "@types/request": "0.0.42",
-    "@types/supertest": "^2.0.0",
-    "@types/uuid": "^3.0.0",
-    "@types/jsonwebtoken": "^7.2.0",
-    "@types/morgan": "^1.7.32",
-    "chai": "^3.0.0",
-    "chai-as-promised": "^6.0.0",
-    "concurrently": "^3.4.0",
-    "mocha": "^3.0.0",
-    "nodemon": "^1.11.0",
-    "nyc": "^10.3.2",
-    "rimraf": "^2.0.0",
-    "source-map-support": "^0.4.15",
-    "supertest": "^3.0.0",
-    "ts-node": "^2.0.0",
-    "tslint": "^4.0.0",
+    "@types/supertest": "2.0.0",
+    "@types/uuid": "3.0.0",
+    "@types/jsonwebtoken": "7.2.0",
+    "@types/morgan": "1.7.32",
+    "chai": "3.5.0",
+    "chai-as-promised": "6.0.0",
+    "concurrently": "3.4.0",
+    "mocha": "3.4.2",
+    "nodemon": "1.11.0",
+    "nyc": "10.3.2",
+    "rimraf": "2.6.1",
+    "source-map-support": "0.4.15",
+    "supertest": "3.0.0",
+    "ts-node": "2.1.2",
+    "tslint": "4.5.1",
     "typescript": "2.3.4",
-    "uuid": "^3.1.0"
+    "uuid": "3.1.0"
   },
   "engines": {
-    "node": ">=4.0.0"
+    "node": "8.1.2"
```

Comment:
Dumb question: We're running our Travis build using [node v7](https://github.com/buildit/bookit-server/blob/package-fixing/.travis.yml#L3). Should the "engine" match that? I'm not sure what "engine" in this context actually does.